build_resolvers/lib/src/resolver.dart

```diff
@@ -451,7 +465,7 @@ final _dartUiPath =
 
 /// The current feature set based on the current sdk version and enabled
 /// experiments.
-FeatureSet _featureSet({List<String> enableExperiments}) {
+FeatureSet _featureSet({List<String>? enableExperiments}) {
   enableExperiments ??= [];
```

Comment:
```suggestion
FeatureSet _featureSet({List<String> enableExperiments = const []}) {
```

We used to use the old pattern as a defensive thing to guard against passing null, but now we can actually be sure we don't get it (well, assuming sound null safety but good enough).